### PR TITLE
Radiomaster RP2 PWM [WIP]

### PR DESCRIPTION
**Pull request by vidma**
unbelievably this works  like a charm and gives two PWM outputs:
- PWM1 instead RX
- PWM2 instead TX

### CODE REVIEW

File: src/hardware/targets.json [hunk 1]
```diff
@@ -1364,6 +1364,14 @@
                 "platform": "esp8285",
                 "firmware": "Unified_ESP8285_2400_RX"
             },
+            "rp2pwm": {
+                "product_name": "RadioMaster RP2 2.4GHz RX PWM",
+                "lua_name": "RM RP2 PWM",
+                "layout_file": "Generic 2400 pwm.json",
```
Comment (vidma) [inline src/hardware/targets.json:1370]: could probably use original file with `overlay` entry? just don't know how to disable/unset:
```
"serial_rx": 3,
    "serial_tx": 1,
```

**Comment by JyeSmith** [comment]
Nice, and it should be easy for any user to mod hardware.html to add PWM outputs.  But Im not sure it needs another target when `Generic 2400 PWMP5` can be flashed to most 2.4 receivers.

**Comment by pkendall64** [comment]
I agree with Jye, we don't need another target for this. Especially when we add in the serial refactor PR that is coming soon.
That will allow a user to choose what pins have what function at runtime.

**Comment by vidma** [comment]
P.S./FYI amazing, also possible to get one more output PWM3 by remapping `pin 0` from button into PWM ! 
`on/off` worked for sure, didn't test PWM. 
`    "pwm_outputs": [1,3, 0],` and removed `button` entry.

**Comment by deadbytefpv** [comment]
https://www.youtube.com/watch?v=JlNLxnsRACE

**Comment by JyeSmith** [comment]
@vidma Im closing this PR for the reasons above.  But keep playing with the outputs, there so much that could be done with even just the single available boot pad on receivers e.g. vtx realpit control, a single cam pan servo etc 😄

**Pull request closed without merging**